tests/integration-tests/reports/reportsApi-test.js

```diff
@@ -70,6 +72,84 @@ const jobPlatform = process.env.JOB_PLATFORM;
                     expect(getReportsResponse.body[0]).to.have.property('is_favorite').and.to.be.equal(true);
                     expect(getReportsResponse.body[0]).to.be.deep.equal(createdReportResponse.body);
                 });
+                it('Run a full cycle with chaos experiments -  should return the single created report with its chaos experiments', async function () {
```

Comment:
In all other cases where there are no exiprements defined, it continues to work well with undefined under experiments property?

Reply:
yes, experiments will not be aded